blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -40,36 +45,36 @@ export class RunDetailsPipeline extends Component {
         }
     }
     componentWillUnmount() {
-        document.removeEventListener('keydown', this._handleKeys);
-        document.removeEventListener('wheel', this._onScrollHandler);
+        document.removeEventListener('scroll', this._onScrollHandler);
     }
- // we bail out on arrow_up key
-    _handleKeys(event) {
-        if (event.keyCode === 38 && this.augmenter.karaoke) {
-            logger.debug('stop follow along by key up');
-            this.augmenter.setKaraoke(false);
+    _onScrollHandler() {
```

Comment:
And then we just handle scrolling instead of mousewheel or keyboard events, and while it doesn't matter, probably makes this compatible with a mobile device